.github/ISSUE_TEMPLATE/issue_template.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Bug/Feature Issue
+about: Use this template for tracking new features and explaining bug
+---
+
+# Summary:
+
+## Expected Behavior:
+
+## Current Behaviour:
+
+## Steps to reproduce:
```

Comment:
This isn't really how we are using issues at this stage of the project now. They are not for bug reporting, but actual project ideas. So this template could either be deleted or be made to encourage contribution on ideas more.

Reply:
Understood. I noticed that too and would proceed with removing this file.